src/platform-implementation-js/dom-driver/gmail/views/gmail-route-view/gmail-route-view.ts

```diff
@@ -432,7 +432,9 @@ class GmailRouteView {
       (event) =>
         !!event.el.querySelector('.if') ||
         !!event.el.querySelector('.PeIF1d') ||
-        !!event.el.querySelector('.a98.iY'),
+        !!event.el.querySelector('.a98.iY') ||
+        (event.el.classList.contains('a98') &&
+          event.el.classList.contains('iY')),
```

Comment:
nit: Could this be simplified to

```suggestion
        !!event.el.querySelector('.a98.iY') ||
        event.el.matches('.a98.iY'),
```

Reply:
Indeed, I did not know this method, thanks 🙏